spec/factories/dishes.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :dish do
+    name "MyString"
+    description "MyString"
+    price 10
+    menu
```

Comment:
Updated Row 6 to `association :menu`
Hopefully this is what you're looking for?